packaging/nnstreamer.spec

```diff
@@ -101,11 +104,6 @@ BuildRequires: ssat
 # For ORC (Oil Runtime Compiler)
 BuildRequires:	pkgconfig(orc-0.4)
 
-%if %{with tizen}
```

Comment:
is this not required now?

Reply:
This `if %{with tizen}` block added `BuildRequires` for tizen sensor framework.
This has been moved to line 82 into the existing `if %{with tizen}` block

Reply:
Ah.. I missed line 71.